### ISSUE TITLE
VSP content feedback - VAOS community care provider selection mockup

### ISSUE THREAD

**Issue by peggygannon** [department-of-veterans-affairs/va.gov-team#15711]
User story
This ticket captures VSP content feedback VAOS community care provider selection flow

Prototype link: Here's the prototype for your review: https://adhoc.invisionapp.com/share/QWZ7WD9GTPJ#/435681239_VAOS-CC-Pick-1

Acceptance criteria

- [x]  VSP/Peggy provides content feedback for the form/tool and any associated static pages
- [x]  Product team incorporates feedback and/or asks follow up questions about the feedback
- [x]  Teams leaves comments about any feedback they decide not to take
- [ ]  Peggy checks form/tool after content feedback has been incorporated

**Comment by peggygannon** [comment]
@outerpress Hello, I'm starting to take a look at this CC mockup and wanted to ask how the *Who do you want to see for this appointment/choose provider* screen related to the design/content that is in this ticket below? 

cc @lenaecb 

https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/10520

**Comment by outerpress** [comment]
@peggygannon the screen you referred to in the mockup would replace the functionality shown in that ticket. Basically we're going to user test:
- Removing the ability to copy/paste in a provider from facility locator
- Adding the ability to add up to three providers by searching directly within the app

**Comment by peggygannon** [comment]
Hello, @outerpress I've dropped in my content recommendations to the mockup (and here below). Let me know if you have any questions or want to discuss any of my recommendations. thx. 


Edits to CC flow mockup

1. landing page

**Request an appointment**

Submit a request for an appointment at a VA medical center, clinic, or approved community care facility.
{button} Request an appointment

-------

 **Choose where you want to receive your care** 
You’re eligible to see either a VA provider or community care provider for this type of care. 

{Radio button} VA medical center or clinic
Choose this option if your preferred provider for this type of care is at a VA health facility.

radio button} Community care facility
Choose this option if you’re not near a VA health facility or if your preferred provider for this type of care is in your local community.

-----------

I feel like we need to set up this screen a little more with content. also, I think the option No/I don't know is getting lost, so I recommend we move that above the option to choose cc providers. 


**Who do you want to see for this  {type of care}**
Do you have a preferred VA-approved community care provider?

{radio button} Yes

{radio button} No/I don't know

{if yes] 
Choose from a list of providers close to you. You can select up to 3 providers. 

{if no} ? What happens is the user chooses No/I don't know?

-------

**Community care preferences**

Do you have a preferred language for your community care provider?

**Comment by outerpress** [comment]
Thank you @peggygannon ! I made most of the changes and/or replied to comments in InVision. 

The only one I didn't get back to was the comment on the provider selection page. Let's chat about that at our design/content sync on Monday.

**Comment by outerpress** [comment]
@PeggyGannon this is my last question from your feedback (I dropped this into an InVision comment too, just didn't want it to get lost):

For the instruction on the "Who do you want to see" page, what about something like: "You can select up to 3 preferred providers. We'll try to schedule with these providers first. If they are unavailable we'll make every effort to find a provider close to your home."

It's still a bit wordy (overusing "provider") but maybe something like that would tee up the interaction better?

**Comment by peggygannon** [comment]
Hi, @outerpress what do you think of this? 

You can select up to 3 providers you'd prefer for this appointment. If they aren’t available, we’ll schedule your appointment with a provider close to your home.

**Comment by outerpress** [comment]
@peggygannon (If it gets confusing capturing this conversation in both InVision and the ticket, let me know):

I made some changes to this page and incorporated your latest suggestion with a few modifications:
- I removed the "I don't know" question; instead, the entire page is optional.
- Added "(Optional)" to the header
- In the main instruction, changed "select"  to "request":
"You can **request** up to 3 providers you'd prefer for this appointment.:

- Added the clarifications to a blue alert box near the navigation:
"If you don’t request a provider, or if your preferred provider isn’t available, we’ll schedule your appointment with a provider close to your home."

I'm hoping (we'll find out in the test) that removing the "I don't know" question reduces the number of decisions veterans have to make on this page, and breaking up the instructions/putting them where needed makes them easier to digest.

Thoughts?

**Comment by peggygannon** [comment]
hello, @outerpress 

Hello, a few more content recommendations: 

1. Can we bump down the point size on (Optional) in the header, to provide some visual separation that it's not part of the head? 


2. slight tweak to this sentence. Since we change the verb to "request," I don't think we need the "prefer" in there anymore. 

You can request up to 3 providers for this appointment

**Comment by outerpress** [comment]
Thanks @peggygannon ! I wasn't able to get this feedback into the prototype before the tests started, but I'm going to link this ticket to the CC Provider MVP design ticket so we can incorporate it there. I'll add you to that ticket too. This one can be closed.

**Comment by peggygannon** [comment]
ok, thank you for the update @outerpress
